docs/posts/signal-number-registration-update.md

```diff
@@ -0,0 +1,42 @@
+---
+date: 2022-11-10
+categories:
+    - Software
+authors:
+    - matchboxbananasynergy
+tags:
+    - PSA
+    - Signal
+    - Instant Messengers
+links:
+    - Signal Configuration Guide: https://www.privacyguides.org/real-time-communication/signal-configuration-hardening/
```

Comment:
this doesn't look right, i think other blog posts use internal links (i.e. `.md` links) for internal related links

Reply:
Yeah but that page isn't on www.privacyguides.org anymore, so it should be:

```suggestion
    - Signal Configuration Guide: signal-configuration-and-hardening.md
```